staticDHCPd/extensions/official/httpdb.py

```diff
@@ -163,14 +176,37 @@ def _lookupMAC(self, mac):
              'error': str(e),
             })
             raise
-            
+
+    def _parse_server_response(self, json_data):
```

Comment:
I think this function should have a different name so its not shared with the global one